### ISSUE TITLE
not working anymore

### ISSUE THREAD

**Issue by ik2wxx** [jamesremuscat/pyze#73]
It's not working anymore, connection issues.
Worked until yesterday.
Maybe on july 1st they have changed something ?

```
Error communicating with Renault API!
{"type":"TECHNICAL","messages":[{"code":"err.tech.wired.kamereon-proxy","message":"Failed to forward request to remote service."}],"errors":[{"errorCode":"err.tech.wired.kamereon-proxy","errorMessage":"Failed to forward request to remote service."}],"error_reference":"TECHNICAL"}
```

**Comment by jamesremuscat** [comment]
OK, so it looks like the "Kamereon token" is not only no longer required, but actively causes their servers to fall over if you include it in a request.

Nothing like making backwards-incompatible and broken changes to your live API that includes a version number to avoid making backwards-incompatible changes to it, eh? :facepalm: 

Fortunately that means the fix is easy... PR shortly.

**Comment by jamesremuscat** [comment]
@ik2wxx Thanks for bringing this to my attention - I've just released `v0.5.0` which includes the required fix.

**Comment by ik2wxx** [comment]
Thank you !!!
Great support !!!
